src/RestaurantButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+// I *think* the function got passed down as props from the parent "Restaurant.js"
+// check with Douglas...!
+function RestaurantButton({ OrderOne }) {
+  return (
+    <button onClick={OrderOne} className="btn btn-primary">
```

Comment:
Yep, this definitely works! Great stuff. It's also good that you've wrapped the `setOrders` function in this function so that it makes more sense in the context of the component the prop is being passed into.